main.js

```diff
@@ -63,8 +68,7 @@ const createRecord = () => {
     process.exit(1);
   }
 
-  console.log(`echo "id=${result.id}" >> $GITHUB_ENV`);
```

Comment:
Based on the `action.yaml` there is no `id` output, but there is a `record_id` output -- updated the output var name to match `record_id`.